astropy/io/fits/tests/test_hdulist.py

```diff
@@ -620,8 +619,7 @@ def test_update_with_truncated_header(self):
         with fits.open(self.temp('temp.fits')) as hdul:
             assert (hdul[0].data == data).all()
 
-    @pytest.mark.xfail(platform.system() == 'Windows',
-                       reason='https://github.com/astropy/astropy/issues/5797')
+    # https://github.com/astropy/astropy/issues/5797
```

Comment:
Is the issue link still needed if this is no longer an issue?

Reply:
I prefer to leave the comment in, in case it comes back to haunt us, because I have no idea why the `xfail` suddenly pass again in this PR when we are not even trying to fix it.

Reply:
I'm adding a one-line comment before that to explain why the issue link is there.